### PR TITLE
refactor account

### PR DESCRIPTION
**Pull request by yaroslavser**

### CODE REVIEW

File: ton_client/src/tvm/call_tvm.rs [hunk 1]
```diff
@@ -16,35 +16,37 @@ use super::types::ResolvedExecutionOptions;
...
+            &mut StackItem::Cell(data),
         )
         .map_err(|err| Error::internal_error(format!("can not put data to registers: {}", err)))?;
 
     let sci = build_contract_info(
         options.blockchain_config.raw_config(),
-        &account.addr,
-        &account.storage.balance,
+        account.get_addr().unwrap(),
+        account.balance().unwrap(),
```
Comment (DezzK) [inline ton_client/src/tvm/call_tvm.rs:49]: Can we be sure that these values are always set?

File: ton_client/src/tvm/call_tvm.rs [hunk 2]
```diff
@@ -86,35 +82,37 @@ pub(crate) fn call_tvm(
                 exception.to_string(),
                 code,
                 Some(exit_arg),
-                &account.addr,
+                &account.get_addr().unwrap(),
                 None,
```
Comment (DezzK) [inline ton_client/src/tvm/call_tvm.rs:86]: And here.

File: ton_client/src/tvm/call_tvm.rs (hunk 1 above)
Comment (yaroslavser) [reply ton_client/src/tvm/call_tvm.rs]: unwrap removed

File: ton_client/src/tvm/call_tvm.rs (hunk 2 above)
Comment (yaroslavser) [reply ton_client/src/tvm/call_tvm.rs:88]: unwrap removed

File: ton_client/src/tvm/call_tvm.rs (hunk 1 above)
Comment (yaroslavser) [reply ton_client/src/tvm/call_tvm.rs]: removed

File: ton_client/src/tvm/call_tvm.rs (hunk 2 above)
Comment (yaroslavser) [reply ton_client/src/tvm/call_tvm.rs:88]: removed

**Pull request merged**